sonar-db/src/main/java/org/sonar/db/issue/IssueDao.java

```diff
@@ -76,20 +76,7 @@ public IssueDto selectOrFailByKey(DbSession session, String key) {
    * <p>Results may be in a different order as input keys (see {@link #selectByOrderedKeys(DbSession, List)}).</p>
    */
   public List<IssueDto> selectByKeys(final DbSession session, List<String> keys) {
-    return DatabaseUtils.executeLargeInputs(keys, new SelectByKeys(mapper(session)));
-  }
-
-  private static class SelectByKeys implements Function<List<String>, List<IssueDto>> {
-    private final IssueMapper mapper;
-
-    private SelectByKeys(IssueMapper mapper) {
-      this.mapper = mapper;
-    }
-
-    @Override
-    public List<IssueDto> apply(@Nonnull List<String> partitionOfKeys) {
-      return mapper.selectByKeys(partitionOfKeys);
-    }
+    return executeLargeInputs(keys, partitionOfKeys -> mapper(session).selectByKeys(partitionOfKeys));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1612)